racket/collects/pkg/private/new.rkt

```diff
@@ -115,8 +115,6 @@ EOS
       (with-output-to-file ".gitignore"
         (lambda () (display #<<EOS
 *~
-\#*
```

Comment:
Why is `\#*` removed? IIUC, that's the format for backup files in DrRacket, and has nothing to do with the issue that @greghendershott was raising.

Reply:
Actually, disregard what I wrote above. `#*` files are actually "autosave files" and they function just like `.#*` for Emacs, so I guess it makes sense to remove them both.